<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_Dialog</class>
 <widget class="QDialog" name="user_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户信息</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>用户信息</string>
   </property>
   <widget class="QLabel" name="user_info_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>571</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="old_password">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>25</y>
      <width>128</width>
      <height>24</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_password">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>128</width>
      <height>24</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirm_password">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>95</y>
      <width>128</width>
      <height>24</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="change_Button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>125</y>
      <width>96</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>原始密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>65</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>新密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码：</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="return_Button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>591</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>当前已借</string>
   </property>
   <widget class="QTreeView" name="borrowedbookview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>570</width>
      <height>214</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>30</number>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
